cime_config/testdefs/testmods_dirs/clm/FatesColdSeedDisp/shell_commands

```diff
@@ -1,9 +1,7 @@
-SRCDIR=`./xmlquery SRCROOT --value`
-CASEDIR=`./xmlquery CASEROOT --value`
-FATESDIR=$SRCDIR/src/fates/
-FATESPARAMFILE=$SRCDIR/fates_params_seeddisp_4x5.nc
-
-ncgen -o $FATESPARAMFILE $FATESDIR/parameter_files/fates_params_default.cdl
+CASE=`./xmlquery CASE --value`
+SRCROOT=`./xmlquery SRCROOT --value`
+FATESDIR=$SRCROOT/src/fates
+FATESPARAMFILE=${SRCROOT}/src/fates/parameter_files/binaries/${CASE}-params.nc
 
 $FATESDIR/tools/modify_fates_paramfile.py --O --fin $FATESPARAMFILE --fout $FATESPARAMFILE --var fates_seed_dispersal_fraction --val 0.2 --allpfts
```

Comment:
Something we wanted to add here was to make sure that ctsm_pylib was loaded, which would be a nice improvement to this. @samrabin worked on this, but had some trouble on izumi. See https://github.com/ESCOMP/CTSM/pull/2336#issuecomment-1939390575
It would be preferred to have that as a part of this, but I'm not sure that's required.

Reply:
I have a branch doing this that I can submit as a PR to this PR's branch.

Reply:
Resolving this since this PR came into the branch...